src/Symfony/Component/Notifier/Bridge/Telegram/TelegramOptions.php

```diff
@@ -112,6 +112,16 @@ public function photo(string $url): static
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function uploadPhoto(string $path): static
```

Comment:
I have to introduce new option, because passing local path to `photo` will break undocumented feature - sending photo by file_id.